cmake/VASTRegisterPlugin.cmake

```diff
@@ -451,6 +500,14 @@ function (VASTRegisterPlugin)
     endif ()
   endif ()
 
+  # Install an example configuration file, if it exists at the plugin project root.
+  if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${PLUGIN_TARGET}.yaml.example")
+    VASTInstallExampleConfiguration(
+      ${PLUGIN_TARGET}
+      "${CMAKE_CURRENT_SOURCE_DIR}/${PLUGIN_TARGET}.yaml.example"
+      "plugin/${PLUGIN_TARGET}.yaml")
+  endif ()
```

Comment:
Instead of doing this automagically, did you consider adding an optional parameter to VASTRegisterPlugin? I.e.:
```
VASTRegisterPlugin(
  TARGET example-analyzer
  ENTRYPOINT example_analyzer.cpp
  # SOURCES
  #   list additional source files here
  TEST_SOURCES tests/example_analyzer.cpp
  SAMPLE_CONFIGURATION example-analyzer.yaml.example)
```

Reply:
I did consider it, but we do it for schemas and integration tests automatically as well so this felt more consistent. If we want to change that, let's change across the board for the next release.

Reply:
Fair enough.